### DISCUSSION TITLE
PrivateGPT output not showing intelligence similar to ChatGPT

### DISCUSSION THREAD

**Discussion by prashant-1926** [imartinez/privateGPT#970] [Q&A]
After using ChatGPT for a while, I was looking for a similar way to interact with my local documents using the power of GPT, so I come across this PrivateGPT and installed this and ingested a total of 150MB of 65 PDFs. 

However, when I queried the PrivateGPT, to my utter disappointment it is simply quoting the text from the PDFs with no human like Generative Intelligence similar to ChatGPT. 

So, I wonder that whether I've missed something during ingestion phase, or is this PrivateGPT is designed to work this way only. 

Request for help and suggestion. 

Thank you.

**Comment by bdutta** [comment]
See https://github.com/imartinez/privateGPT/discussions/241
